### ISSUE TITLE
Correctly set XP version in jar file

### ISSUE THREAD

**Issue by alansemenov** [enonic/xp-gradle-plugin#4]
It seems that now minVersion and maxVersion are hardcoded to "6.0.0" and "7.0.0". We should set minVersion from ``${xpVersion}`` in build.gradle file. We don't really need maxVersion.

**Comment by sigdestad** [comment]
Is this related to the jar file? I thought that what was listed in the  UI simply never checked the jar file?

**Comment by alansemenov** [comment]
No, it's wrong in jar (and hardcoded in gradle plugin)

**Comment by sigdestad** [comment]
Ups.. But I thought installing new apps on old XP's would fail? Doesn't it?

**Comment by alansemenov** [comment]
Not sure - never tried it with XP < 6.0

**Comment by sigdestad** [comment]
An app that requires 6.12 should not be possible to install and run on 6.11

**Comment by alansemenov** [comment]
I guess you are talking about the list of Market apps in the Install dialog where we filter out apps based on the version. But version in the jar file is something different. We have min version and max version there and as of now they are hardcoded to 6.0 and 7.0.

**Comment by aro** [comment]
Fixed, pending review.

When using the new release of the plugin, the version needs to be added to build.gradle, using the `systemVersion` property:

```
app {
    name = 'com.enonic.app.myapp'
    displayName = 'My App'
    vendorName = 'Enonic AS'
    vendorUrl = 'http://enonic.com'
    systemVersion = "${xpVersion}"   <--
}
```

If it is missing the build fails and there is an error indicating were to set the missing property in build.gradle.

**Comment by alansemenov** [comment]
Released in version 1.0.15 of the gradle plugin
